src/browser-capabilities.ts

```diff
@@ -36,26 +38,31 @@ const browserPredicates: {
     es2015: since(49),
     push: since(41),
     serviceworker: since(45),
+    modules: since(61),
```

Comment:
I think we'll need either more fidelity than this, or plan to update these later on, to capture `import()` support and `import.meta`, which we'll need for PRPL. See https://www.chromestatus.com/features/5684934484164608

I think I'm ok with just planning on updating these values...